doc/source/whatsnew/v1.3.2.rst

```diff
@@ -23,6 +23,7 @@ Fixed regressions
 - Fixed regression where :meth:`pandas.read_csv` raised a ``ValueError`` when parameters ``names`` and ``prefix`` were both set to None (:issue:`42387`)
 - Fixed regression in comparisons between :class:`Timestamp` object and ``datetime64`` objects outside the implementation bounds for nanosecond ``datetime64`` (:issue:`42794`)
 - Fixed regression in :meth:`.Styler.highlight_min` and :meth:`.Styler.highlight_max` where ``pandas.NA`` was not successfully ignored (:issue:`42650`)
+- Fixed regression in :meth:`pandas.concat` where copy=False was not honored in ``axis=1`` Series concatenation. (:issue:`42501`)
```

Comment:
```suggestion
- Fixed regression in :func:`pandas.concat` where ``copy=False`` was not honored in ``axis=1`` Series concatenation (:issue:`42501`)
```

Reply:
I committed the suggestion.  Thanks for spotting that.